cfe_internal/enterprise/federation/federation.cf

```diff
@@ -162,6 +166,11 @@ bundle agent transport_user
           expression => strcmp("Enforcing", execresult("$(paths.getenforce)", useshell));
 
     enabled.selinux_enabled::
+        # expectation is that for el6 hosts the Makefile will be present and must be used
```

Comment:
Should we specify that using `if =>"redhat_6";`?

Or, should this depend on the promise for the makefile being present?

It's nice when the model works globaly, and is easy to read.